docs/notebooks/topic_labeling.ipynb

```diff
@@ -0,0 +1,125 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**initial setup**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import gensim\n",
+    "import os\n",
+    "import itertools\n",
+    "\n",
+    "wiki_path = 'D:\\gensim_data\\simplewiki'\n",
```

Comment:
You can generate sample based on https://github.com/RaRe-Technologies/gensim-data/releases/tag/wiki-english-20171001 (for reproducibility)